fastlane/Fastfile

```diff
@@ -36,6 +41,25 @@ end
 default_platform(:ios)
 
 platform :ios do
+  # Testing
+
+  desc 'Build for testing'
+  lane :build_for_testing do
```

Comment:
@llleyelll I think we can use these lanes for CI files also, please help me to update the CI files

Reply:
@suho Do you mean to update this approach to using those lanes? Am I understand it correctly? 😅
```
- name: Build and Test
  run: xcodebuild -workspace {workspace_name}.xcworkspace -scheme '{staging_scheme_name}' -destination 'platform=iOS Simulator,name=iPhone 12' clean test | xcpretty && exit ${PIPESTATUS[0]}
```

Reply:
Yes, as we are using `fastlane scan` for testing, so we can update it for the `Build and Test` steps.
I prefer we have some think like this
```rb

lane :build_and_test do
  build_for_testing
  tests
  ui_tests
end
```

then:
```yml
- name: Build and Test
  run: bundle exec fastlane build_and_test
```

Reply:
Updated in [d5dc2a9](https://github.com/nimblehq/ios-templates/pull/170/commits/d5dc2a9ac1ba7bc3dc2db5d54a375b4e675a0298) 🙏